tests/override-v12.nix

```diff
@@ -0,0 +1,14 @@
+{pkgs ? import <nixpkgs> {
+    inherit system;
+}, system ? builtins.currentSystem}:
+
+let
+  nodePackages = import ./default-v12.nix {
+    inherit pkgs system;
+  };
+in
+nodePackages // {
+  floomatic = nodePackages.floomatic.override {
+    buildInputs = [ pkgs.pkgconfig pkgs.qt4 ];
+  };
+}
```

Comment:
can you please explain how is this test testing the change made in PackageExpression.js? I don't think I get it. Thanks

Reply:
I didn't add any tests to check changes, I just run old tests. This file makes old tests run, because there is currently an error `.... file not found`

Reply:
gotcha, now it makes more sense, thanks.